recipes/thrax/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "thrax" %}
+{% set version = "1.3.1" %}
+
+package: 
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://www.opengrm.org/twiki/pub/GRM/ThraxDownload/{{ name }}-{{ version }}.tar.gz
+  sha256: d16a24376c8e722b694825366eda8a8accfd7515abb7a3db563c19056f951203
+  
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - openfst ==1.7.5
+  run:
+    - openfst ==1.7.5
```

Comment:
Same, does this need such a tight pinning?

Reply:
In this case, yes. (I am an upstream maintainer for both OpenFst and Thrax.) Basically every release of OpenFst involves massive API/ABI breaking, if only by nature of it being mostly a template library. (Upstream also has what are probably thousands of test units but they're proprietary to Google.)